config/samples/csiaddons_v1alpha1_networkfenceclass.yaml

```diff
@@ -6,4 +6,7 @@ metadata:
     app.kubernetes.io/managed-by: kustomize
   name: networkfenceclass-sample
 spec:
-  # TODO(user): Add fields here
+  provisioner: driver.csi.com
```

Comment:
Don't use `csi.com` (it is a valid domain), use `.example.com` (or `.io`, `.net`, ..) instead